tests/performance/reindexing/reindexing_and_feeding.rb

```diff
@@ -5,150 +5,112 @@
 require 'performance_test'
 require 'app_generator/search_app'
 
-class ReindexingAndFeeding < PerformanceTest
+class ReindexingAndFeedingTest < PerformanceTest
 
   def initialize(*args)
     super(*args)
   end
 
   def timeout_seconds
-    900
+    1800
   end
 
   def setup
-	  super
-	  set_description("Measure throughput of reindexing, and its impact on external updates and puts")
-	  set_owner("jvenstad")
-	  @test_config = {
-		  {
-			  :legend => 'Reindexing',
-			  :fbench => { :clients => 4, :use_post => true },
-			  :data => "{ \"fields\": { \"text\": \"GNU's not UNIX\" } }"
-		  } => {
-			  'qps' => { :y_min =>  2650, :y_max =>  3250 },
-			  '95p' => { :y_min =>   1.4, :y_max =>   1.7 }
-		  },
-		  {
-			  :legend => 'GET small data 4 clients',
-			  :fbench => { :clients => 4, :use_post => false }
-		  } => {
-			  'qps' => { :y_min =>  5600, :y_max =>  6250 },
-			  '95p' => { :y_min =>   0.7, :y_max =>   0.9 }
-		  },
-		  {
-			  :legend => 'POST large data 128 clients',
-			  :fbench => { :clients => 128, :use_post => true},
-			  :data => "{ \"fields\": { \"text\": \"GNU#{"'s not UNIX" * (1 << 10) }\" } }"
-		  } => {
-			  'qps' => { :y_min =>  6300, :y_max =>  6700 },
-			  '95p' => { :y_min =>    24, :y_max =>    28 }
-		  },
-		  {
-			  :legend =>  'GET large data 128 clients',
-			  :fbench => { :clients => 128, :use_post => false }
-		  } => {
-			  'qps' => { :y_min => 29500, :y_max => 33000 },
-			  '95p' => { :y_min =>   4.9, :y_max =>   5.5 }
-		  }
-	  }
-	  @graphs = get_graphs
-	  @app = SearchApp.new.monitoring("vespa", 60).
-		  container(Container.new("combinedcontainer").
-			    jvmargs('-Xms16g -Xmx16g').
-			    search(Searching.new).
-			    docproc(DocumentProcessing.new).
-			    gateway(ContainerDocumentApi.new)).
-	  admin_metrics(Metrics.new).
-	  indexing("combinedcontainer").
-	  sd(selfdir + "doc.sd")
+    super
+    set_description("Measure throughput of reindexing, and its impact on external updates and puts")
+    set_owner("jvenstad")
   end
 
   def test_reindexing_performance_and_impact
+    @graphs = get_graphs(graph_config)
+
+    @app = SearchApp.new.monitoring("vespa", 60).
+      container(Container.new("combinedcontainer").
+		jvmargs('-Xms16g -Xmx16g').
+		search(Searching.new).
+		docproc(DocumentProcessing.new).
+		gateway(ContainerDocumentApi.new)).
+    admin_metrics(Metrics.new).
+    indexing("combinedcontainer").
+    sd(selfdir + "doc.sd")
+
     deploy_app(@app)
     start
 
+    @qrserver = @vespa.container["combinedcontainer/0"]
+    @document_count = 1 << 19
```

Comment:
I prefer an explicit number here.

Reply:
I like powers of two, and so does bucket distribution, but OK :)